doc/android-build.md

```diff
@@ -0,0 +1,37 @@
+# Building binaries of LibIndy for Android
+
+**Not ready for production use! Not fully tested.**
+
+## Prerequisites
+
+- Docker
```

Comment:
Can i build it on windows and MacOS?

Reply:
Only Linux is supported as of now.

Reply:
Can you add this info to per-requirements? It is better to provide info about platform that we tested.

Reply:
That info is present in _Known Issues_ segment at the bottom of the document.

Reply:
ok